help/whats_new/_posts/2019-02-01-whats-new.md

```diff
@@ -3,6 +3,39 @@ This page is updated monthly with a list of new (**[new]**) and improved (**[imp
 
 To receive this listing via monthly email newsletter, along with announcements and new blog posts, [sign up here](http://eepurl.com/ggoxhn).
 
+## May 2024
+* Announcements:
+   + We are extremely pleased to introduce our very first cohort of NASA SciX Ambassadors! These exceptional individuals, representing diverse research disciplines, will play an important role in expanding the reach and impact of NASA SciX by serving as liaisons between the platform and their communities. Through their work, they will enhance the interdisciplinary spirit of NASA SciX and work with us to advance the cause of open science. Learn more about each of the new [NASA SciX Ambassadors here](https://www.scixplorer.org/scixblog/announcing-2024-ambassadors).
```

Comment:
Use the relative path on this link (`blog/announcing-2024-ambassadors`) instead of the full link - it's safer in case we change URLs in the future. Also, I'd use `blog` in the link (as I have it) instead of `scixblog` - the What's New page only exists in the ADS side of things, not the SciX side, so good to keep it consistent.